corehq/apps/dashboard/templates/dashboard/dashboard_new_user.html

```diff
@@ -21,6 +21,37 @@
 </script>
 {% endblock js-inline %}
 
+{% block uncompressed-js %} {{ block.super }}
+    <script type="text/javascript">
+        (function (angular, undefined) {
+            'use strict';
+            var dashboardApp = angular.module('dashboardApp', []);
```

Comment:
While this is ok to do everything in the page for an a/b test (as the code is temporary) if code grows at all or will have long term support, it's better to add it in an external file. Best practices for doing so are outlined VERY well in this video, and I highly recommend you give it the hour to watch (or speed it up 2x for a half hour): https://www.youtube.com/watch?v=i9MHigUZKEM

For now inline JS is ok (as it's an a/b test). But please put a comment at the top that this style should not be copied in the rest of the code base.